scheduler/remote_server.py

```diff
@@ -0,0 +1,487 @@
+import inotify.adapters
```

Comment:
Would be nice to have a description of what this code does at the top. Writes atq commands, updates automatically using inotify based on the scd file

Reply:
done